### PR TITLE
Add missing "Escapers" title in the book

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
Some reading experience improvement. :)

### CODE REVIEW

**Comment by GuillaumeGomez** [comment]
I also added an example because the explanation was definitely unclear.

File: book/src/configuration.md [hunk 1]
```diff
@@ -125,3 +127,11 @@ first escaper configured and ending with the default escapers for HTML
 (no escaping; `md`, `yml`, `none`, `txt`, and the empty string). Note that
 this means you can also define other escapers that match different extensions
 to the same escaper.
+
+As an example, we want `.js` files to be treated like "txt" files. To do so:
+
+```toml
+[[escaper]]
+path = "js"
```
Comment (Kijewski) [inline book/src/configuration.md:135]: At the end of the week, my brain is a mush right now, so maybe I'm misunderstanding something, but this does not look right.

Comment (GuillaumeGomez) [reply book/src/configuration.md:135]: It's how I use it though. XD

**Comment by GuillaumeGomez** [comment]
Fixed the completely wrong original update.

File: book/src/configuration.md [hunk 2]
```diff
@@ -118,10 +120,35 @@ extensions = ["tex"]
...
 (no escaping; `md`, `yml`, `none`, `txt`, and the empty string). Note that
 this means you can also define other escapers that match different extensions
 to the same escaper.
+
+You can then use templates with this extension or use the
+[`escape`](https://docs.rs/rinja/latest/rinja/filters/fn.escape.html) filter with
+the name of your extension in your template:
+
+```jinja
+{% some_string|escape("tex")
```
Comment (Kijewski) [inline book/src/configuration.md:139]:
```suggest
{{ some_string|escape("tex") }}
```

Comment (GuillaumeGomez) [reply book/src/configuration.md]: 👍

**Review by Kijewski — APPROVED** [review]
:+1:

**Pull request merged**